frontend/src/Editor/ActionTypes.js

```diff
@@ -24,7 +24,8 @@ export const ActionTypes = [
     name: 'Go to app',
     id: 'go-to-app',
     options: [
-      { name: 'app', type: 'url', default: 'https://app.tooljet.io/applications/app-id' }
+      { name: 'app', type: 'url', default: 'https://app.tooljet.io/applications/app-id' },
```

Comment:
`url` is not needed, right?

Reply:
Correct, changed it to `text` and defaulted to `''`.